xla/service/gpu/triton_autotuner.cc

```diff
@@ -638,6 +731,17 @@ CompileMany(const AutotuneConfig& config, AutotunerCompileUtil& util,
         log(has_executable.value());
         counter.DecrementCount();
       });
+
+      if (IsFusionKind(*fusion, kCuDnnFusionKind) ||
+          (IsFusionKind(*fusion, kTritonGemmFusionKind) &&
+           EnableCuDnn(config, debug_opts))) {
+        for (const int64_t plan_id : GetCuDnnPlanIds(*fusion, config)) {
```

Comment:
Are all available plans being used for auto-tuning? Would it make sense to impose a limit to reduce the impact on compile time?

Reply:
Only few of them do compile, other ones exit early without much impact on compilation time. This may change in future versions of cuDNN.

Added a limit.